Bottom: 17d044354330f9c34c095790b6232712f8a59540
Top:    6fb8573194dbc9f8ef55057e36e76ca4fc63bb13
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-02-16 23:51:41 +0000

Adjust STEP export to 0.2mm board thickness (Hack - need something quick!)


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index a9065ab..a96459f 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -60,7 +60,7 @@
 #endif
 
 
-#define HACK_BOARD_THICKNESS MM_TO_COORD(2.3)
+#define HACK_BOARD_THICKNESS MM_TO_COORD(0.2)
 #define HACK_MASK_THICKNESS MM_TO_COORD(0.01)
 
 static GList *object3d_test_objects = NULL;
diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index 02e9528..8150c46 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -45,7 +45,7 @@
 
 #define CRASH fprintf(stderr, "HID error: pcb called unimplemented STEP function %s.\n", __FUNCTION__); abort()
 
-#define HACK_BOARD_THICKNESS MM_TO_COORD(2.3)
+#define HACK_BOARD_THICKNESS MM_TO_COORD(0.2)
 
 
 HID step_hid;
